### PR TITLE
Add Ukrainian localisation for UI app, Product App, Booking App

### PR DESCRIPTION
**Pull request by daniloff200**
## Issue Reference
#6893 

## Description
Add Ukrainian 🇺🇦  local apps

### CODE REVIEW

File: packages/Webkul/BookingProduct/src/Resources/lang/uk/app.php [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                'from' => 'Від',
+                'to' => 'До',
+                'qty' => 'Кількість',
+                'add-slot' => 'Додати слот',
+                'sunday' => 'Неділя',
+                'monday' => 'Понеділок',
+                'tuesday' => 'Вівторок',
+                'wednesday' => 'Середа',
+                'thursday' => 'Четвер',
+                'friday' => 'П'ятниця',
```
Comment (lex111) [inline packages/Webkul/BookingProduct/src/Resources/lang/uk/app.php:33]: Note that using the single quote within a string enclosed in single quotes requires escaping via backslash like that:

```suggestion
                'friday' => 'П\'ятниця',
```

The same is true for other similar messages.

**Comment by daniloff200** [comment]
@lex111  Thanks, I've updated it

**Comment by Sanjaybhattwebkul** [comment]
Hi there,
Getting **syntax error**, please find the **screenshot** [Here](https://prnt.sc/QyOoOuU16-cm)

**Comment by daniloff200** [comment]
hi @Sanjaybhattwebkul , thanks for finding that, I've fixed the issue

**Comment by Sanjaybhattwebkul** [comment]
Hi there,
**Still not fixed,** getting the **syntax error** again on the **same file**. please find the screenshot [Here](https://prnt.sc/VsK18ZZilt92)

**Comment by daniloff200** [comment]
@Sanjaybhattwebkul  not sure, how it happened, I checked the code validation, it was fine. But, I've added few changes here, should be fine

**Comment by Sanjaybhattwebkul** [comment]
@daniloff200 
**Thanks** for your **contribution**.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**